scripts/build.sh

```diff
@@ -2,6 +2,9 @@
 
 set -ev
 
+export BUILD_VERSION="0.0.2-SNAPSHOT"
```

Comment:
Yes, automatic would be better.
We should discuss how we want this to work for the whole project.